chargeOclock/devTemp/ClientTextFieldDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ClientTextFieldDelegate.swift
+//  chargeOclock
+//
+//  Created by Andrej Krizmancic on 20/02/2017.
+//  Copyright © 2017 CocoaHeads Slovenia. All rights reserved.
+//
+
+import UIKit
+
+class ClientTextFieldDelegate: NSObject, UITextFieldDelegate {
```

Comment:
not sure I'd create a separate class just for this... and even if, it would need a better name, since the current name doesn't give a clue as to what it does (except that it's a text field delegate)